### PR TITLE
Update instruction for upgrading to ASM 1.13 in KF 1.5.1 for GKE

### PR DESCRIPTION
**Pull request by gkcalat**
- Updating ASM upgrading instructions for GKE
- Version bump to 1.5.1

### CODE REVIEW

File: content/en/docs/distributions/gke/deploy/upgrade.md [hunk 1]
```diff
@@ -258,6 +260,8 @@ Kubeflow on Google Cloud doesn't guarantee the upgrade for each Kubeflow compone
 
 ### Upgrade Kubeflow cluster to v1.5
 
+Starting from Kubeflow v1.5.1 we upgraded ASM to v1.13. Please, follow the instructions on how to [upgrade ASM (Anthos Service Mesh)](#upgrade-asm-anthos-service-mesh). If you want to use ASM version prior to 1.11, refer to [the legacy instructions](https://github.com/kubeflow/gcp-blueprints/blob/master/kubeflow/common/asm/deprecated/README.md).
```
Comment (zijianjoy) [inline content/en/docs/distributions/gke/deploy/upgrade.md:263]: Using the term `Please` here shows politeness but audiences are looking for instruction but not communication. We can omit the `Please` here and directly say `Follow the instructions ...`.

File: layouts/shortcodes/gke/latest-version.html [hunk 2]
```diff
@@ -1 +1 @@
-1.5.0
\ No newline at end of file
+1.5.1
```
Comment (zijianjoy) [inline layouts/shortcodes/gke/latest-version.html:1]: We need to wait for this release to be made on https://github.com/kubeflow/gcp-blueprints/releases before merging this doc change.

File: content/en/docs/distributions/gke/deploy/upgrade.md (hunk 1 above)
Comment (gkcalat) [reply content/en/docs/distributions/gke/deploy/upgrade.md]: Fixed

File: content/en/docs/distributions/gke/deploy/upgrade.md [hunk 3]
```diff
@@ -24,6 +24,8 @@ This guide assumes the following settings:
 
 ## General upgrade instructions
 
+Starting from Kubeflow v1.5.1 we upgraded ASM to v1.13, which comes with a slightly different upgrade process. Please, follow instructions on how to [upgrade ASM (Anthos Service Mesh)](#upgrade-asm-anthos-service-mesh)
```
[the remark this answers is not in the document]
Comment (gkcalat) [reply content/en/docs/distributions/gke/deploy/upgrade.md]: Fixed

**Review by Arhell — COMMENTED** [review]
/lgtm

**Comment by zijianjoy** [comment]
/lgtm

Thank you Ablai! Once we have released 1.5.1, I will approve this PR.

**Pull request merged**